packages/bitcoinml/bitcoinml.0.2.4/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "Davide Gessa <gessadavide@gmail.com>"
+authors: "Davide Gessa <gessadavide@gmail.com>"
+homepage: "https://github.com/dakk/bitcoinml"
+bug-reports: "https://github.com/dakk/bitcoinml/issues"
+license: "MIT"
+dev-repo: "https://github.com/dakk/bitcoinml.git"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "jbuilder" {build & >= "1.0+beta11"}
+  "base" {build & >= "v0.9.2"}
+  "stdio" {build & >= "v0.9.0"}
+  "configurator" {build & >= "v0.9.1"}
+  "bitstring" {build & >= "2.1.0"}
+  "ppx_jane" {build & >= "v0.9.0"}
+  "ppx_bitstring" {build & >= "2.0.0"}
+  "bignum" {build & >= "v0.9.0"}
+  "cryptokit" {build & >= "1.11"}
+  "stdint" {build & >= "0.3.0-0"}
```

Comment:
@dakk, could you remove the `build` annotation from the dependencies that the library uses at runtime (e.g. `stdint`, `cryptokit`, probably others).  `build` is for dependencies like `jbuilder` that are only used to build the package and are not needed to use the library.

No need to open a new request -- you can push changes directly to this branch, or run `opam-publish submit` again.